doc/gen_docs.go

```diff
@@ -39,6 +45,177 @@ func main() {
 	// wg.Wait()
 }
 
+// If a flag is not listed in flagInfos, docs will generate the default values for the flag:
+// flagName = lowercase name
+// defaultVal = $({uppercase name})
+// paramConflicts = none
+var flagInfos = map[string]BridgeFlagInfo{
+	"BaseURL": {
+		flagName:       "base-url",
+		defaultVal:     "$(BASE_URL)",
+		paramConflicts: []string{},
+	},
+	"CredPrefix": {
+		flagName:   "credential",
+		defaultVal: "$(CREDENTIALS)",
+		paramConflicts: []string{
+			"TokenRaw",
+		},
+	},
+	"TokenRaw": {
+		flagName:       "token",
+		defaultVal:     "$(TOKEN)",
+		paramConflicts: []string{},
+	},
+}
+
+type BridgeFlagInfo struct {
+	flagName       string
+	defaultVal     string
+	paramConflicts []string
+}
+
+var bridgeUrls = map[string]string{
+	"github":            "https://github.com/MichaelMure/git-bug",
+	"gitlab":            "https://gitlab.com/gitlab-org/gitlab",
+	"launchpad-preview": "https://bugs.launchpad.net/ubuntu/",
+	// TODO: Insert URL for Jira Project
+}
+
+// genBridgeConfig generates the bridge configuration documentation on go generate
+// Documentation is stored in commands/bridge_configure_doc.go
+func genBridgeConfig() error {
+	var exampleText strings.Builder
+	exampleText.WriteString("`")
+	exampleText.WriteString(`# Interactive example
+[1]: github
+[2]: gitlab
+[3]: jira
+[4]: launchpad-preview
+
+target: 1
+name [default]: default
+
+Detected projects:
+[1]: github.com/a-hilaly/git-bug
+[2]: github.com/MichaelMure/git-bug
+
+[0]: Another project
+
+Select option: 1
+
+[1]: user provided token
+[2]: interactive token creation
+Select option: 1
+
+You can generate a new token by visiting https://github.com/settings/tokens.
+Choose 'Generate new token' and set the necessary access scope for your repository.
+
+The access scope depend on the type of repository.
+Public:
+	- 'public_repo': to be able to read public repositories
+Private:
+	- 'repo'       : to be able to read private repositories
+
+Enter token: 87cf5c03b64029f18ea5f9ca5679daa08ccbd700
+Successfully configured bridge: default
+
+`)
+	targets := bridge.Targets()
+	for i, b := range targets {
+		if i != 0 {
+			exampleText.WriteString("\n\n")
+		}
+		exampleText.WriteString("# For ")
+		exampleText.WriteString(strings.Title(b))
+		exampleText.WriteString("\ngit bug bridge configure \\\n")
+
+		exampleText.WriteString("    --target=")
+		exampleText.WriteString(strings.ToLower(b))
+		exampleText.WriteString(" \\\n")
+
+		exampleText.WriteString("    --url=")
```

Comment:
--url is not meant to be a flag that exist in all bridges so it should probably be deal with below. Also ideally it would fallback to the default $(URL) placeholder but that starts to be really really nitpicking

Reply:
ha well, --url is not supported by the Jira bridge

Reply:
it's this one: https://github.com/MichaelMure/git-bug/blob/master/bridge/core/params.go#L8